source/vibe/appmain.d

```diff
@@ -65,6 +65,13 @@ int main()
 		} else {
 			status = runEventLoop();
 		}
+
+		
+		version(VibeLibasyncDriver) {
+			import libasync.threads : destroyAsyncThreads;
+			destroyAsyncThreads(); // destroy threads
+		}
```

Comment:
It can only be destroyed once and for all actually. I should probably look into that more